### ISSUE TITLE
Pack Manager: multithreading for parallel pack detection

### ISSUE THREAD

**Issue by stu85010** [gnu-mcu-eclipse/eclipse-plugins#314]
The pack fetching process costing a lots of time (hours) and sometimes stuck at some unreachable pack, because some server is down or very slow.

It would be nice if we could have parallel process to download the pack. Because most of the commonly used pack and server is very fast.

I'm using the version:
GNU MCU C/C++ Packs (Experimental)	2.2.2.201801250917

**Comment by ilg-ul** [comment]
You are right, pack fetching is slow, and the various servers not very reliable.

There are some possible workarounds, but I had no time to consider them.
